ustriage/ustriage.py

```diff
@@ -754,6 +756,13 @@ def launch():
                         dest='extended_format',
                         help='Do Display bugs in extended format which adds'
                              ' date-last-updated, importance and assignee')
+    parser.add_argument('-A', '--age',
+                        default=-1,
+                        type=int,
+                        dest='age',
+                        help='Mark bugs older than this many days (default'
```

Comment:
This seems to be the opposite of what the flag actually does...

Reply:
you are right, fixed